app/uk/gov/hmrc/eoricommoncomponent/frontend/forms/SubscriptionForm.scala

```diff
@@ -53,11 +53,14 @@ object SubscriptionForm {
 
   private def validSicCode: Constraint[String] =
     Constraint("constraints.sic")({
-      case s if s.trim.isEmpty       => Invalid(ValidationError("cds.subscription.sic.error.empty"))
-      case s if !s.matches("[0-9]*") => Invalid(ValidationError("cds.subscription.sic.error.wrong-format"))
-      case s if s.length < 4         => Invalid(ValidationError("cds.subscription.sic.error.too-short"))
-      case s if s.length > 5         => Invalid(ValidationError("cds.subscription.sic.error.too-long"))
-      case _                         => Valid
+      case s if s.filterNot(_.isWhitespace).isEmpty => Invalid(ValidationError("cds.subscription.sic.error.empty"))
```

Comment:
Maybe we could we trim out the white space and store that in a variable, instead of having to duplicate the validation on each case?

We also need some unit tests to cover this, drop me a ping and we can go through it together if needed 👍🏻